matRad_calcPhotonDose.m

```diff
@@ -91,63 +91,49 @@
 [yCoordsV_vox, xCoordsV_vox, zCoordsV_vox] = ind2sub(ct.cubeDim,V);
 
 % set lateral cutoff value
-lateralCutoff = 82; % [mm]
+if ~(strcmp(num2str(pln.bixelWidth),'field'))
+    lateralCutoff = 82; % [mm]
+else
+    lateralCutoff = 140; % [mm] due to the large field size
+end
 
 % toggle custom primary fluence on/off. if 0 we assume a homogeneous
 % primary fluence, if 1 we use measured radially symmetric data
 useCustomPrimFluenceBool = 0;
 
+% set skin treshhold in relative electron density to water, which 
+% determines the start and stop points for dose calculation
+skinThreshold = 0;
+
 %% kernel convolution
-% load polynomial fits for kernels ppKernel1, ppKernel2, ppKernel3
+% prepare data for convolution to reduce calculation time
 fileName = [pln.radiationMode '_' pln.machine];
 try
    load([fileparts(mfilename('fullpath')) filesep fileName]);
 catch
    error(['Could not find the following machine file: ' fileName ]); 
 end
 
-% Make a 2D grid extending +/-100mm with 0.1 mm resolution
+% Make a 2D grid extending +/-100mm with 0.5 mm resolution
```

Comment:
Ok so this reduction in resolution was for consistency with PDC - any idea what kind of loss in accuracy that means? In particular for small fields during bixel dose calculations?